api-catalog-ui/frontend/src/components/Wizard/configs/wizard_base_categories.jsx

```diff
@@ -205,10 +245,12 @@ export const baseCategories = [
             keyStorePassword: {
                 value: '',
                 question: 'The password used to unlock the keystore:',
+                type: 'password',
             },
             keyStoreType: {
-                value: '',
+                value: 'PKCS12',
                 question: 'Type of the keystore:',
```

Comment:
Just a small note:  to be consistent with the truststore label: either, either change this to `question`: 'Keystore type:'` or the other way around